src/timer.cpp

```diff
@@ -87,4 +87,9 @@ void reset_timers()
   simulation::time_event_death.reset();
 }
 
+void restart_timers()
+{
+  simulation::time_total.start();
```

Comment:
I think a better solution than adding a `restart_timers` function is to just have `openmc_init` and `openmc_run` start _and_ stop the `time_total` timer.

Reply:
Thanks for your comments. However, depletion calls `openmc_init` only once (`init`->`run`->`reset`->`run`->...).  If I start and stop `time_total` timer in `openmc_init` and `openmc_run`, the timer only counts the total time for the first batch of neutron transport. (`start`->`stop`->`reset`->`stop`->`reset`->...) All zeros for other batches.